<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Video Tools</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Extract Clips</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QCheckBox" name="outputSubdirectories1Cbx">
         <property name="text">
          <string>Output To Subdirectories</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QToolButton" name="output1Btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="maxLengthSbx"/>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="extractSingle1Btn">
         <property name="text">
          <string>Extract Single</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Single Video</string>
         </property>
         <property name="buddy">
          <cstring>singleVideo1Led</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Output</string>
         </property>
         <property name="buddy">
          <cstring>output1Led</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Directory</string>
         </property>
         <property name="buddy">
          <cstring>directory1Led</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Crop Variation</string>
         </property>
         <property name="buddy">
          <cstring>cropVariation1Sbx</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="fpsSbx"/>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="extractDirectory1Btn">
         <property name="text">
          <string>Extract Directory</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="splitCutsCbx">
         <property name="text">
          <string>Split At Cuts</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="directory1Led"/>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="removeBorders1Cbx">
         <property name="text">
          <string>Remove Borders</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="singleVideo1Led"/>
       </item>
       <item row="2" column="4">
        <widget class="QToolButton" name="directory1Btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="output1Led"/>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="singleVideo1Btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Max Length (s)</string>
         </property>
         <property name="buddy">
          <cstring>maxLengthSbx</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="cropVariation1Sbx"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Time Range</string>
         </property>
         <property name="buddy">
          <cstring>timeRangeStart1Led</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="timeRangeStart1Led">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>99:99:99</string>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="timeRangeStop1Led">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>99:99:99</string>
         </property>
         <property name="text">
          <string>99:99:99</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Set FPS</string>
         </property>
         <property name="buddy">
          <cstring>fpsSbx</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Extract Images</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="4">
        <widget class="QPushButton" name="extractDirectory2Btn">
         <property name="text">
          <string>Extract Directory</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="cropVariation2Sbx"/>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="extractSingle2Btn">
         <property name="text">
          <string>Extract Single</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="directory2Btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Crop Variation</string>
         </property>
         <property name="buddy">
          <cstring>cropVariation2Sbx</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="removeBorders2Cbx">
         <property name="text">
          <string>Remove Borders</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Images/sec</string>
         </property>
         <property name="buddy">
          <cstring>imagesSecSbx</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="output2Btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Output</string>
         </property>
         <property name="buddy">
          <cstring>output2Led</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="singleVideo2Led"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="output2Led"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Single Video</string>
         </property>
         <property name="buddy">
          <cstring>singleVideo2Led</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Time Range</string>
         </property>
         <property name="buddy">
          <cstring>timeRangeStart2</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="singleVideo2Btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="timeRangeStop2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>99:99:99</string>
         </property>
         <property name="text">
          <string>99:99:99</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Directory</string>
         </property>
         <property name="buddy">
          <cstring>directory2Led</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Blur Removal</string>
         </property>
         <property name="buddy">
          <cstring>blurRemovalSbx</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="outputSubdirectories2Cbx">
         <property name="text">
          <string>Output To Subdirectories</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="timeRangeStart2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>99:99:99</string>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="directory2Led"/>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="imagesSecSbx"/>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="blurRemovalSbx"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Download</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Single Link</string>
         </property>
         <property name="buddy">
          <cstring>singleLinkLed</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="singleLinkLed"/>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="downloadLinkBtn">
         <property name="text">
          <string>Download Link</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Link List</string>
         </property>
         <property name="buddy">
          <cstring>linkListLed</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="linkListLed"/>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="linkListBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="downloadListBtn">
         <property name="text">
          <string>Download List</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Output</string>
         </property>
         <property name="buddy">
          <cstring>output3Led</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="output3Led"/>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="output3Btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Additional Args</string>
         </property>
         <property name="buddy">
          <cstring>additionalArgsTed</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QPlainTextEdit" name="additionalArgsTed">
         <property name="plainText">
          <string>--quiet --no-warnings --progress</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="infoBtn">
         <property name="text">
          <string>yt-dlp Info</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>singleVideo1Led</tabstop>
  <tabstop>singleVideo1Btn</tabstop>
  <tabstop>extractSingle1Btn</tabstop>
  <tabstop>timeRangeStart1Led</tabstop>
  <tabstop>timeRangeStop1Led</tabstop>
  <tabstop>directory1Led</tabstop>
  <tabstop>directory1Btn</tabstop>
  <tabstop>extractDirectory1Btn</tabstop>
  <tabstop>output1Led</tabstop>
  <tabstop>output1Btn</tabstop>
  <tabstop>outputSubdirectories1Cbx</tabstop>
  <tabstop>splitCutsCbx</tabstop>
  <tabstop>maxLengthSbx</tabstop>
  <tabstop>fpsSbx</tabstop>
  <tabstop>removeBorders1Cbx</tabstop>
  <tabstop>cropVariation1Sbx</tabstop>
  <tabstop>singleVideo2Led</tabstop>
  <tabstop>singleVideo2Btn</tabstop>
  <tabstop>extractSingle2Btn</tabstop>
  <tabstop>timeRangeStart2</tabstop>
  <tabstop>timeRangeStop2</tabstop>
  <tabstop>directory2Led</tabstop>
  <tabstop>directory2Btn</tabstop>
  <tabstop>extractDirectory2Btn</tabstop>
  <tabstop>output2Led</tabstop>
  <tabstop>output2Btn</tabstop>
  <tabstop>outputSubdirectories2Cbx</tabstop>
  <tabstop>imagesSecSbx</tabstop>
  <tabstop>blurRemovalSbx</tabstop>
  <tabstop>removeBorders2Cbx</tabstop>
  <tabstop>cropVariation2Sbx</tabstop>
  <tabstop>singleLinkLed</tabstop>
  <tabstop>downloadLinkBtn</tabstop>
  <tabstop>linkListLed</tabstop>
  <tabstop>linkListBtn</tabstop>
  <tabstop>downloadListBtn</tabstop>
  <tabstop>output3Led</tabstop>
  <tabstop>output3Btn</tabstop>
  <tabstop>additionalArgsTed</tabstop>
  <tabstop>infoBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
